src/m365/planner/commands/task/task-get.ts

```diff
@@ -0,0 +1,61 @@
+import { Logger } from '../../../../cli';
+import { CommandOption } from '../../../../Command';
+import GlobalOptions from '../../../../GlobalOptions';
+import request from '../../../../request';
+import { GraphItemsListCommand } from '../../../base/GraphItemsListCommand';
+import commands from '../../commands';
+import { Task } from '../../Task';
+
+interface CommandArgs {
+  options: Options;
+}
+
+interface Options extends GlobalOptions {
+  taskId: string;
+}
+
+class PlannerTaskGetCommand extends GraphItemsListCommand<Task> {
+  public get name(): string {
+    return commands.TASK_GET;
+  }
+
+  public get description(): string {
+    return 'Retrieve the details of the specified planner task';
+  }
+
+  public defaultProperties(): string[] | undefined {
+    return ['id', 'description'];
+  }
+
+  public commandAction(logger: Logger, args: CommandArgs, cb: () => void): void {
+
+    const requestOptions: any = {
+      url: `https://graph.microsoft.com/v1.0/planner/tasks/${args.options.taskId}/details`,
```

Comment:
Since `taskId` is provided by the user we need to escape it before passing it to the query